website/docs/r/schema_site_service_graph.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: "mso"
+page_title: "MSO: mso_schema_site_service_graph"
+sidebar_current: "docs-mso-resource-schema_site_service_graph_node"
+description: |-
+  Manages MSO Schema Site Level Service Graph Node
+---
+
+# mso_schema_site_service_graph #
+
+Manages MSO Schema Site Level Service Graph Node.
+
+## Example Usage ##
+
+```hcl
+
+resource "mso_schema_site_service_graph" "site_service_graph" {
+  schema_id          = mso_schema_site.schema_site_1.schema_id
+  site_id            = mso_schema_site.schema_site_1.site_id
+  template_name      = mso_schema_template_service_graph.test_sg.template_name
+  service_graph_name = mso_schema_template_service_graph.test_sg.service_graph_name
+  service_node {
+    device_dn = data.aci_l4_l7_device.l4_l7_device_1.id
+  }
+  service_node {
+    device_dn = data.aci_l4_l7_device.l4_l7_device_2.id
+  }
+}
+
+```
+
+## Argument Reference ##
+* `schema_id` - (Required) The schema ID under which you want to deploy Service Graph.
+* `template_name` - (Required) The template name under which you want to deploy Service Graph.
+* `site_id` - (Required) The site ID under which you want to deploy Service Graph.
+* `service_graph_name` - (Required) The name of the Service Graph.
+* `service_node` - (Required) List of maps to provide Site level Node association. This maps should be provided if site is associated with template.
```

Comment:
Not sure I understand the second part, "This maps should be provided if site is associated with template.". Because the list of maps and the attribute inside the map are both required.

Reply:
changed.